google-apis-common/src/field_mask.rs

```diff
@@ -34,6 +34,12 @@ fn snakecase(source: &str) -> String {
 #[derive(Clone, Debug, Default, PartialEq, Eq)]
 pub struct FieldMask(Vec<String>);
 
+impl FieldMask {
+    pub fn new<S: AsRef<str>>(values: &[S]) -> Self {
```

Comment:
The public API should be documented as it [goes up on docs.rs](https://docs.rs/google-apis-common/latest/google_apis_common/field_mask/struct.FieldMask.html).

Reply:
Added!